### DISCUSSION TITLE
Unable to load bet365

### DISCUSSION THREAD

**Discussion by fabinhoshz** [cefsharp/CefSharp#3377] [General]
Hi, 

I'm using CEFSharp for long time but today I has a little problem that I cant find what is the error... I'm work with a specific website named bet365.com and since today the site dont load in my browser. Just load a blank page.

If anyone can help me, to try access and check too. Maybe can be a new resource or feature to add in project.

Thanks!

**Comment by amaitland** [comment]
What version are you using? If you are using an old version I'd suggest testing version 86 or the 87 pre release. 

I'd also suggest you check the log file see https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file

**Comment by fabinhoshz** [comment]
I'm in 86, the least version, pls help me found what is the problem

________________________________

**Reply by amaitland** [reply]
Did you check the log file?

**Comment by fabinhoshz** [comment]
Yes, but log file seems normal, I'm using a long time and never has some problem, today start but just this website (bet365.com), I cant found the problem, in Chrome (For Windows) is fine.

I check in devtools that has a function (rest get) that not complete, still 'pending' for a long time and after I get a catch inside devtools showing that is paused to avoid memory-out exception.

This site is working until hours ago :(

If you can help me, if you has a miminal example with you try navigate to you show.

Thanks!
________________________________

**Reply by amaitland** [reply]
I'd suggest testing with the CEF Sample application see https://github.com/cefsharp/CefSharp/blob/master/CONTRIBUTING.md#cefsharp-vs-chromium-embedded-frameworkcef for details. 

You can test with a newer version of CEF  see https://cef-builds.spotifycdn.com/index.html#windows32 for download links.

**Comment by fabinhoshz** [comment]
Hi Alex,

I make test but is same, the cef is freeze at get bet365.com url...

[cid:fd577b22-1e85-46a8-bfaf-60a66eb8717e]
________________________________

**Comment by student712** [comment]
Having the same issue with the same site. I can get slightly further by disabling the following setting: browserSettings.LocalStorage, but still no luck in getting the page to load. 

I downloaded the the example at https://github.com/cefsharp/CefSharp.MinimalExample and changed nothing apart from editing the url on line 23 of BrowserForm.cs and that won't even display the site. I've added `settings.LogSeverity = LogSeverity.Verbose;` to Program.cs in that project and am seeing this in the console:

```
[0130/165932.988:INFO:crash_reporting.cc(220)] Crash reporting enabled for process: browser
[0130/165933.027:VERBOSE1:pref_proxy_config_tracker_impl.cc(186)] 00000232C6A31880: set chrome proxy config service to 00000232C6A3D1C0
[0130/165933.044:VERBOSE1:webrtc_internals.cc(118)] Could not get the download directory.
[0130/165933.238:VERBOSE1:pref_proxy_config_tracker_impl.cc(186)] 00000232C8FBB370: set chrome proxy config service to 00000232C6B5DF20
```
I tried visiting the site in both the WPF/OffScreen and WinForms versions of the CEF Sample application mentioned above (and also the newest version of CEF), but neither work either.

**Comment by fabinhoshz** [comment]
Hi,

Me too, I'm since yesterday but no success :(

I create exacly request headers, include cookies used by native Chrome, so many UA (like Chrome too), I'm using pre-release 87 fo cefSharp too.

Its possible see that one call is for long time 'Pending' and after 30~40s is stopped to avoid out-of-memory exception, I'll send a print-screen to show...

This call: https://www.bet365.com/defaultapi/sports-configuration

[cid:29fdf6ad-6d06-48cd-ab99-8001f385aad6]

I has a project that is entire developed based in CEFSharp to use bet365.com, if anyone can help us with this, I can make a donate to thanks!

Thank you.

________________________________

**Comment by AlissonMarcel** [comment]
I have this same problem using CEF4Delphi, which is also based on CEF. Any discovery around here, I warn you!

**Reply by amaitland** [reply]
Sounds a lot like https://magpcss.org/ceforum/viewtopic.php?f=6&t=18142 

As this isn't CefSharp specific I'd suggest you continue the conversation there.

**Comment by fabinhoshz** [comment]
The bet365.com can read some property that shows access by CEFSharp and not Chrome.

Anyone know if exists some property to set name or attribute equals Chrome?

Thanks

**Comment by lijing2001** [comment]
I think that Bet365's javascript blocked cefsharp loaded, I have to use cefSettings.CefCommandLineArgs.Add("disable-javascript", "true") to Initialized, I can load background part of bet365 page

**Reply by fabinhoshz** [reply]
Hi, yes, I'm testing here yet and with JS disabled its possible see something of website.

I think that inside JS has some 'if' that can check if webdriver version is cefsharp :(

Thanks!

**Reply by emanlee6688** [reply]
Do you find some clue??

**Reply by nil-nothing1024** [reply]
The same question.Does anyone have solution to solve it?Thanks.

**Comment by emanlee6688** [comment]
I have found that not only cefsharp, but also eo.webbrowser, which is the same as cefsharp, chromenium embedded have the same problem. There must be some kind of characters that make embedded-chromenium different from chrome.
somebody has clues?

**Reply by BRKind** [reply]
Did you find anything. I'm willing to pay someone to tell me

**Comment by helderppb** [comment]
Does anyone have found the solution for this issue? I have the same problem.

**Comment by willowTank** [comment]
Any news?

**Comment by ns-jisorce** [comment]
See https://github.com/berstend/puppeteer-extra/tree/master/packages/puppeteer-extra-plugin-stealth
